### ISSUE TITLE
Making one-off schedules easier to add

### ISSUE THREAD

**Issue by maweki** [nielsfaber/scheduler-card#538]
### Checklist

- [X] I checked for similar existing requests (both open and closed) before posting.
- [X] My request is generic, other users may benefit from it too.
- [X] I will participate in further discussion about this feature and can test it (if requested) once it's done.

### Proposal

I mostly use Scheduler to create one-off schedules. For example when I am cooking with the slow cooker overnight and I want the cooker to start at 3 in the morning or be off at 8.

Currently I need to:

- Select the switch and the action (obviously, this can't be avoided)
- Select a "daily" time, even though it's not gonna be daily
- Select "options"
- Select "delete"
- Save (obviously, this can't be avoided)

Now the card doesn't show that the schedule is gonna be deleted. It just shows "daily"

### Additional info

I think besides selecting "daily", "working day", "weekend" and "custom"(? I have the german localization) there could be a "once" selection which has precisely the semantics to run the schedule once at the next occurence of the selected time and delete it after, no further setup necessary.

In the overview screen of the card there should then be the info "once" (instead of daily). I think in general the "delete after" state should be shown in the overview, but that's another issue.

**Comment by nielsfaber** [comment]
"once" is not a valid day, so I don't agree it should become an option for the day selection.
If you don't want/need to specify a day, you can leave it untouched. The default setting is 'daily' which seems appropriate for your usage.

The 'delete after completion' is considered an _option_ (which a lot of people don't use), therefore I think it is located fine as it is.
Adding an option 'once' to the days selection will raise a lot of confusion for users.
Also the scheduler component treats this completely different than the loop behaviour (behaviour after completion).
In summary, I don't think your request is a good improvement.
